### ISSUE TITLE
[Bug]: undefined symbol: __libc_vfork

### ISSUE THREAD

**Issue by fleschutz** [snapcrafters/jenkins#33]
### What happened?

I have set up a fresh Jenkins snap with recommended plugins on Ubuntu 22.04, x86_64. I start a simple Pipeline job and receive the following error:
...
Caused by: hudson.plugins.git.GitException: Command "git fetch --tags --force --progress -- https://github.com/naiveskill/devops.git +refs/heads/*:refs/remotes/origin/*" returned status code 128:
stdout: 
stderr: /usr/lib/git-core/git-remote-https: symbol lookup error: /snap/core18/current/lib/x86_64-linux-gnu/libpthread.so.0: undefined symbol: __libc_vfork, version GLIBC_PRIVATE
...
Any hint what is wrong?

### What should have happened?

No error.

### Output of `snap info $snap_name`

```shell
➤ sudo snap info jenkins
name:      jenkins
summary:   Jenkins automation server
publisher: Snapcrafters
store-url: https://snapcraft.io/jenkins
contact:   https://github.com/snapcrafters/jenkins/issues
license:   unset
description: |
  Continuous integration and delivery server with a vast ecosystem of plugins

  -  Continuous Integration and Continuous Delivery

      As an extensible automation server, Jenkins can be used as a simple CI server or turned into
      the continuous delivery hub for any project.

  -  Easy configuration

     Jenkins can be easily set up and configured via its web interface, which includes on-the-fly
     error checks and built-in help.

  - Plugins

     With hundreds of plugins in the Update Center, Jenkins integrates with practically every tool
     in the continuous integration and continuous delivery toolchain.

  -  Extensible

     Jenkins can be extended via its plugin architecture, providing nearly infinite possibilities
     for what Jenkins can do.

  -  Distributed

     Jenkins can easily distribute work across multiple machines, helping drive builds, tests and
     deployments across multiple platforms faster.

  This snap is a `classic` snap. If you're running an older release of this snap then you need to
  `snap refresh jenkins --classic` to refresh into classic builds.

  This snap is maintained by the Snapcrafters community, and is not necessarily endorsed or
  officially maintained by the upstream developers.
commands:
  - jenkins.config
services:
  jenkins: simple, enabled, active
snap-id:      TKjv5xnnKfr2MgxFRoWWNqHuQCJOrIU4
tracking:     latest/edge
refresh-date: today at 16:47 CEST
channels:
  latest/stable:    2.290 2021-04-29 (2126) 131MB classic
  latest/candidate: ↑
  latest/beta:      2.325 2021-12-19 (2767) 131MB classic
  latest/edge:      2.337 2022-03-06 (2978) 152MB classic
installed:          2.337            (2978) 152MB classic
```

### Output of `snap connections $snap_name`

```shell
output is empty
```

### Output of `snap version`

```shell
➤ sudo snap version
snap    2.58.3
snapd   2.58.3
series  16
ubuntu  22.04
kernel  5.15.0-52-generic
```

### Relevant log output

```shell
Caused by: hudson.plugins.git.GitException: Command "git fetch --tags --force --progress -- https://github.com/naiveskill/devops.git +refs/heads/*:refs/remotes/origin/*" returned status code 128:
stdout: 
stderr: /usr/lib/git-core/git-remote-https: symbol lookup error: /snap/core18/current/lib/x86_64-linux-gnu/libpthread.so.0: undefined symbol: __libc_vfork, version GLIBC_PRIVATE
```

**Comment by fleschutz** [comment]
I have used "sudo snap install jenkins --edge --classic" for installation.

**Comment by fleschutz** [comment]
Fixed by using the latest Jenkins 2.433 Snap (released yesterday). Thank you very much!
